the-collab-lab/tcl-34-smart-shopping-list
# Additem Page

_For an example of how to fill this template out, [see this Pull Request]

## Description

This PR adds the new layout to the Additem screen. To maintain the standard, between the two main screens (List and AddItem)

## Related Issue

Closes #35

## Acceptance Criteria

The user wants to be able to see the application from his cell phone and also from his laptop without affecting the design of the application

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
| ✓   | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

![image](https://user-images.githubusercontent.com/74724204/142671254-97d891a0-3c0d-434f-b9b8-5638ce766992.png)


### After

![image](https://user-images.githubusercontent.com/74724204/142671513-44ea0d75-b2c1-4638-a130-c40708baafe6.png)

![image](https://user-images.githubusercontent.com/74724204/142671649-0f835df8-5194-486a-ac09-0d1585a42412.png)



## Testing Steps / QA Criteria

From your terminal, pull down this branch with git pull additem-view and check that branch out with git checkout additem-view
Update dependencies: npm install react-icons --save
npm start to launch the app
Fill in the "Shared token" field with the isa-collection token
Access the additem screen
